src/NewCommand.php

```diff
@@ -89,6 +90,9 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $process->run(function ($type, $line) use ($output) {
             $output->write($line);
         });
+        
+        $this->replaceDefaultDatabase("{$directory}/.env", $name);
+        $this->replaceDefaultDatabase("{$directory}/.env.example", $name);
```

Comment:
Is it necessary to replace default database on `.env.example` file?

The `.env.example` is about the sample `.env` file and I think it doesn't have to replace the default with current database name.

Reply:
The whole idea of this PR is to save time. The `.env` file is [ignored](https://github.com/laravel/laravel/blob/master/.gitignore#L6) by default, so every time someone clones the git repository of a project, they copy `.env.example` to `.env`. Not updating `.env.example` still requires that you manually update the database in `.env`.

Reply:
@pascalbaljet , thank you for your reply. That makes sense.

Reply:
I don't think it makes sense to copy this information into .env.example. .env.example is not gitignored by default. This would increase the risk of users pushing internal configuration into git repositories.

It wouldn't be an insane risk because it's "only" the database name, but it would still be unnecessary.